site/src/pages/TerminalPage/TerminalPage.tsx

```diff
@@ -67,6 +67,16 @@ const TerminalPage: FC = () => {
   const config = useQuery(deploymentConfig());
   const renderer = config.data?.config.web_terminal_renderer;
 
+  // Periodically report workspace usage.
+  useQuery(
+    workspaceUsage({
+      usageApp: "reconnecting-pty",
+      terminal,
+      workspaceId: workspace.data?.id,
+      agentId: workspaceAgent?.id,
+    })
+  );
```

Comment:
Shouldn't we only need to do this if we have the experiment enabled?

Reply:
I made the decision for clients to send the request indiscriminately because I wanted the experiment to control the server behavior and not the clients. This way when the experiment is promoted to GA we don't have to worry about out of date clients dropping data. That said, for browsers this window is very small and easy to update with a refresh - but I still chose to do this on the web client to have the same behavior on all clients for a more predictable design. 

Do you see any issues with this happening on non-experiment installations? It would make it so the `last_used_at` is bumped since that's the endpoints behavior with no experiment enabled, but I see this as an improvement in coverage for that.

Reply:
My only concern would be the added load, but I don't think that would be a huge increase given that the built-in terminal is basically your "last-ditch" option for accessing the workspace.

Reply:
It's also on a 1 minute interval so it's not too noisy of an interval